core/modules/czar/Czar.cc

```diff
@@ -29,29 +29,25 @@
 
 // Third-party headers
 #include "boost/lexical_cast.hpp"
-#include "boost/property_tree/ini_parser.hpp"
-#include "boost/property_tree/ptree.hpp"
 
 // LSST headers
 #include "lsst/log/Log.h"
 
 // Qserv headers
 #include "ccontrol/ConfigMap.h"
 #include "czar/MessageTable.h"
+#include "util/ConfigStore.h"
 #include "util/IterableFormatter.h"
 
 namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.Czar");
 
-// read configuration file
-lsst::qserv::StringMap readConfig(std::string const& configPath);
-
 // parse KILL query, return thread ID or -1
 int parseKillQuery(std::string const& query);
 
 // make mysql config object from config map
-lsst::qserv::mysql::MySqlConfig mysqlConfig(lsst::qserv::StringMap const& config);
+lsst::qserv::mysql::MySqlConfig mysqlConfig(lsst::qserv::util::ConfigStore const& config);
```

Comment:
I'd make this method either a constructor of the `MySqlConfig` class or its static method.